docs/posts/wow-analysis/index.qmd

```diff
@@ -0,0 +1,212 @@
+---
+title: "Analsis of World of Warcraft Data"
```

Comment:
```suggestion
title: "Analysis of World of Warcraft Data"
```

I think this may have gotten clobbered in a subsequent push!

Reply:
Thank you! Just fixed, hopefully for the last time on this specific typo haha.